components/tabstrip/scrollable-tabs.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Scrollable Tabs
+page_title: TabStrip Scrollable Tabs
+description: Scrollable Tabs in the TabStrip for Blazor.
+slug: tabstrip-scroll-tabs
```

Comment:
scroll-tabs may be harder to remember and use without checking. Why not use scrollable-tabs, as the article title?